server/cmake/modules/FindLibClang.cmake

```diff
@@ -61,20 +61,20 @@ endforeach()
 
 find_path(LIBCLANG_INCLUDE_DIR clang-c/Index.h
   PATHS ${libclang_llvm_header_search_paths}
-  PATH_SUFFIXES LLVM/include #Windows package from http://llvm.org/releases/
+  PATH_SUFFIXES llvm/include #Windows package from http://llvm.org/releases/
```

Comment:
Sorry, I should add some motivation for the change as well -- on my OS X machine, it was indeed discovering a `libclang.dylib` library at `/usr/local/llvm/lib/libclang.dylib`, but persisted the capital `LLVM` despite that the directory was indeed `llvm`. This might affect users with case-sensitive filesystems (although CMake might just automatically normalize the path in that case, I am not sure)